### DISCUSSION TITLE
How to avoid Contacts duplicate by Social security number (SSN)

### DISCUSSION THREAD

**Discussion by espe808** [YetiForceCompany/YetiForceCRM#14745] [How to]
Like VAT ID to avoid Accounts duplicate, it is possible to do this also for the Contacts creating, for example, a field like Social security number (SSN) unique for each Contact and so avoiding to have duplicate?

**Comment by vovpff** [comment]
You need to write handler what will check SSN and prevent duplicating

**Reply by mariuszkrzaczkowski** [reply]
ideally it was for the handler to have the configuration and then you could choose the field after which it checks for duplicates, it takes 4-8 hours

**Reply by espe808** [reply]
Thanks for the answer.
And so I need to add custom code to do this or there is an option in the configuration panel to add the function to a field like the SSN?

**Reply by vovpff** [reply]
You need to write custom handler. As Marius said you can charge for 4-8 hours support and devs implement more flexible solution.

**Reply by espe808** [reply]
How can I charge the support?

**Reply by vovpff** [reply]
Check official website or home page of admin panel

**Reply by espe808** [reply]
Ok thanks
